docs/source/compatibility.rst

```diff
@@ -105,97 +91,95 @@ and how users enable or disable the warnings practically.
 Supported Backward Compatibility
 --------------------------------
 
-This section defines backward compatibilities that minor updates must maintain.
+This section defines backward compatibilities that revision updates must maintain.
 
 Documented Interface
 ~~~~~~~~~~~~~~~~~~~~
 
 Chainer has the official API documentation.
 Many applications can be written based on the documented features.
 We support backward compatibilities of documented features.
-In other words, codes only based on the documented features run correctly with minor/revision-updated versions.
+In other words, codes only based on the documented features run correctly with revision-updated versions.
 
 Developers are encouraged to use apparent names for objects of implementation details.
 For example, attributes outside of the documented APIs should have one or more underscores at the prefix of their names.
 
+.. note::
+
+   Although it is not stated as a rule, we also try to keep the compatibility for any interface that looks like a stable feature.
+   For example, if the name of a symbol (function, class, method, attribute, etc.) is not prefixed by an underscore and the API is not experimental,
+   the API should be kept over revision updates even if it is not documented.
+
 .. _undocumented_behavior:
 
 Undocumented behaviors
 ~~~~~~~~~~~~~~~~~~~~~~
 
 Behaviors of Chainer implementation not stated in the documentation are undefined.
-Undocumented behaviors are not guaranteed to be stable between different minor/revision versions.
+Undocumented behaviors are not guaranteed to be stable between different revision versions.
 
-Minor update may contain changes to undocumented behaviors.
-For example, suppose an API X is added at the minor update.
-In the previous version, attempts to use X cause AttributeError.
-This behavior is not stated in the documentation, so this is undefined.
-Thus, adding the API X in minor version is permissible.
```

Comment:
It seems this example is still valid if we replace "minor update" with "revision update". So I don't think we need to remove it.